src/lib/import-export/import/importers/importer.ts

```diff
@@ -72,53 +72,14 @@ abstract class BaseImporter extends EventEmitter implements Importer {
 			}
 		}
 
-		await this.replaceSiteUrl( siteId );
+		await updateSiteUrlToLocal( siteId );
 		this.emit( ImportEvents.IMPORT_DATABASE_COMPLETE );
 	}
 
 	protected async prepareSqlFile( _tmpPath: string ): Promise< void > {
 		// This method can be overridden by subclasses to prepare the SQL file before import.
 	}
 
-	protected async replaceSiteUrl( siteId: string ) {
```

Comment:
Moved it outside as a separate function since now it's used not only for import flow, so it's better to move it outside of import/export modules.